api/Cargo.toml

```diff
@@ -10,11 +10,12 @@ license = "Apache-2.0 OR BSD-3-Clause"
 [dependencies]
 lazy_static = "1.4.0"
 log = "0.4.8"
-micro_http = { git = "https://github.com/cloud-hypervisor/micro-http.git", branch = "master" }
+micro_http = { git = "https://github.com/openanolis/dbs-uhttp", rev = "52e7ffe994134c231ad9fe1cc88c68b0cdda2278" }
```

Comment:
@xujihui1985 @jiangliu could you release `dbs-uhttp` on cargo.io?

Reply:
We are waiting for a confirmation signal to publish "dbs-uhttp". Is it ready now?

Reply:
@jiangliu It's ready, test for macos is ok.